compiler-core/src/erlang/tests.rs

```diff
@@ -632,3 +632,19 @@ pub fn main() {
 "
     );
 }
+
+// https://github.com/gleam-lang/gleam/issues/3304
+#[test]
+fn type_named_else() {
```

Comment:
Another test for functions too please

Reply:
`else` is a keyword in gleam as well so we can't have a function named like that

Reply:
oh yes!!!